com.jeffser.Alpaca.AMD.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "com.jeffser.Alpaca.AMD",
+    "branch": "1.0",
+    "runtime": "com.jeffser.Alpaca",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk//46",
+    "build-extension": true,
+    "build-options": {
+        "prefix": "/app"
+    },
+    "cleanup": [
+        "*.a",
+        "*.la"
+    ],
+    "modules": [
+        {
+            "name": "rocm",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -r --remove-destination lib/ollama/* ${FLATPAK_DEST}/"
+            ],
+            "only-arches": [
+                "x86_64"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/ollama/ollama/releases/download/v0.3.9/ollama-linux-amd64-rocm.tgz",
+                    "sha256": "792b27c731334c377679b6ec6580f4b77833b495a3932be6c4398b4f9bbb80fa"
+                }
+            ]
+        },
+        {
+            "name": "metainfo",
```

Comment:
Add this to the rocm module as a post-install command. No need for a separate module for a single command.